### PR TITLE
Add homarr-iframes project note in the iframes widget docs

### PR DESCRIPTION
**Pull request by diogovalentte**
> I don't know if the iframe widget page is the best place to put it, or if the note I added is too big, so you can change it if you decide to add it.

### Screenshot
image

### CODE REVIEW

File: docs/widgets/iframe/index.mdx [hunk 1]
```diff
@@ -60,3 +60,12 @@ Check out our documentation on [editing tiles](/docs/getting-started/after-the-i
 
 ## Screenshots
 ![](./widget.webp)
+
+---
+
+## homarr-iframes
```
Comment (manuel-rw) [inline docs/widgets/iframe/index.mdx:66]: Please adjust the title structure:

```
## Third party content
### homarr-iframes from YOUR_NAME
```

Comment (manuel-rw) [inline docs/widgets/iframe/index.mdx:66]: Add an info using this tool  https://docusaurus.io/docs/markdown-features/admonitions and mention that this is third party content and all support requests / problems must be redirected to the appropriate repository.

File: docs/widgets/iframe/index.mdx [hunk 2]
```diff
@@ -60,3 +60,12 @@ Check out our documentation on [editing tiles](/docs/getting-started/after-the-i
...
+The project [homarr-iframes](https://github.com/diogovalentte/homarr-iframes) has iframes designed to be used with Homarr. These iframes offer a minimalist view of other systems, like viewing your tasks from [Vikunja](https://github.com/go-vikunja/vikunja) or bookmarks from [Linkwarden](https://github.com/linkwarden/linkwarden).
```
Comment (manuel-rw) [inline docs/widgets/iframe/index.mdx:67]: There is some ballast here.
Remove ``other systems, like`` and just make a bullet point list with the apps supported and what you can do with them:
```md
- go-vikunja: Do X, Y and Z
- linkwarden: Do U and Z
```

File: docs/widgets/iframe/index.mdx [hunk 3]
```diff
@@ -60,3 +60,12 @@ Check out our documentation on [editing tiles](/docs/getting-started/after-the-i
...
+
+![](./homarr-iframes-example.webp)
```
Comment (manuel-rw) [inline docs/widgets/iframe/index.mdx:69]: Add an alternative text to this image. Can be useful for blind people and screen readers.

File: docs/widgets/iframe/index.mdx [hunk 4]
```diff
@@ -60,3 +60,12 @@ Check out our documentation on [editing tiles](/docs/getting-started/after-the-i
...
+
+The iframes have options for configuration, like selecting the iframe theme and limit, and other options specific to each iframe.
```
Comment (manuel-rw) [inline docs/widgets/iframe/index.mdx:71]: Perhaps link to these options or the documentation about them?

**Review by manuel-rw — CHANGES_REQUESTED** [review]

**Comment by diogovalentte** [comment]
Thanks for your feedback, I updated the docs.

**Review by ajnart — APPROVED** [review]
LGTM ! Cool project 😎

**Pull request merged**